assets/styles/authoring/advanced-authoring.scss

```diff
@@ -458,22 +460,14 @@ $font-family-sans-serif: var(--font-sans);
       position: relative;
     }
   }
-  .aa-sequence-editor {
-    max-height: 60vh;
-    min-height: 30vh;
+  .aa-sequence-editor-menu {
+    overflow-y: auto;
   }
-
-  .aa-adaptivity-rules {
+  .aa-adaptivity-editor .aa-sequence-editor {
     max-height: 30vh;
   }
   .aa-sequence-editor,
   .aa-adaptivity-rules {
-    overflow-y: auto;
```

Comment:
Reverted some changes that was done for MER-1880. There are pending issue in that ticket so I will work on that ticket in different PR.